### ISSUE TITLE
[Bug] Right-click doesn't work on a Chrome window

### ISSUE THREAD

**Issue by hrbtk** [domferr/tilingshell#97]
**Describe the bug**
Right-click doesn't have abillity to snap Chrome window

**To Reproduce**
1. Open Chrome
2. Right click on window title

**Screenshots**
![image](https://github.com/user-attachments/assets/54277a5b-e729-4e35-858e-8021db2a3887)
![image](https://github.com/user-attachments/assets/aef0829f-e5d7-43c3-b3f2-b70a0bf038a1)

**Information (please complete the following):**
 - Tiling Shell version: 19
 - GNOME version: 46.3.1 (Fedora 40)

**Additional context**
Google Chrome: Version 126.0.6478.182 (Official Build) (64-bit)

**Comment by thyttan** [comment]
My chromium browser (not chrome) has a "Use system titlebar and borders" checkbox. 

![image](https://github.com/user-attachments/assets/5978edcf-a0b3-4ae5-a8ca-dcf4028d67bb)

And if I do right click on that I get the system window menu:

![image](https://github.com/user-attachments/assets/b8edbe78-ced3-4b47-a72d-8ed0f6f85cc2)

**Comment by domferr** [comment]
Yeah, unfortunately some apps override the system context menu when you right click...
If they do that, we cannot add the snap features in any way.

Thank you @thyttan for sharing! Folks using firefox can right-click _twice_ to show the system context menu.
